src/05_lists.py

```diff
@@ -8,22 +8,26 @@
 
 # Change x so that it is [1, 2, 3, 4]
 # YOUR CODE HERE 
-print(x)
+print(x.append(4))
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
 # YOUR CODE HERE 
-print(x)
+print(x.extend(y))
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
 # YOUR CODE HERE 
-print(x)
+print(x.pop(4 & 5))
 
 # Change x so that it is [1, 2, 3, 4, 9, 99, 10]
 # YOUR CODE HERE 
-print(x)
+z = [99, 10]
+x.pop(5)
+print(x.extend(z))
 
```

Comment:
This could also be done using insert with index and number params, Ex: `x.insert(5, 99)`